modules/switches.py

```diff
@@ -0,0 +1,93 @@
+import random
+import modules
+
+
+class Switches(modules.Module):
+    identifiers = ['switches']
+    display_name = "Switches"
+    manual_name = "Switches"
+    help_text = "`{cmd} flip 1 5 3 4 2 5 2` or `{cmd} flip 1534252`. Partial solutions allowed."
+    module_score = 1
+
+    def __init__(self, bomb, ident):
+        super().__init__(bomb, ident)
+
+        self.invalidPositions = "11011/10100/10000/01101/01100/01000/00111/00101/00011/00001".split("/")
```

Comment:
This looks like a constant. Use `UPPER_SNAKE_CASE` and define it in the class itself. Also, use a list - that's clearer and takes few seconds more to write (`:s@/@", "@` in vim, for example).

Reply:
Actually, use a set - `{...}` instead of `[...]`. The order is irrelevant and `x in a_set` is faster than `x in a_list`.